src/test/java/uk/gov/companieshouse/pscdataapi/api/ChsKafkaApiServiceTest.java

```diff
@@ -39,6 +53,9 @@ class ChsKafkaApiServiceTest {
 
     private static final String EVENT_TYPE_CHANGED = "changed";
     private static final String EVENT_TYPE_DELETED = "deleted";
+    private static final String DELTA_AT = "20240219123045999999";
+    private static final String PSC_URI = "/company/%s/persons-with-significant-control/"
+            + "%s/%s";
```

Comment:
String concatenation not needed here. Can be on one line.

Reply:
done